connector/connect/common/src/main/protobuf/spark/connect/ml.proto

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = 'proto3';
+
+package spark.connect;
+
+import "spark/connect/relations.proto";
+
+option java_multiple_files = true;
+option java_package = "org.apache.spark.connect.proto";
+
+
+message RemoteCall {
+  oneof call_type {
+    ConstructObject construct_object = 1;
+    DestructObject destruct_object = 2;
+    CallMethod call_method = 3;
+    CallFunction call_function = 4;
+  }
+
+  message ArgValue {
+    oneof arg_type {
+      int32 int32_value = 1;
+      int64 int64_value = 2;
+      float float_value = 3;
+      double double_value = 4;
+      bool bool_value = 5;
+      string string_value = 6;
+      List list = 7;
+      Map map = 8;
+      Relation relation = 9;
+      RemoteObject remote_object = 10;
```

Comment:
Note: I will add `Matrix` and `Vector` message type field into oneof field of `ArgValue` later. Some ML methods return matrix or vector objects.